JMMServer/Providers/MyAnimeList/MALHelper.cs

```diff
@@ -530,15 +532,21 @@ public static void UpdateMALSeries(AnimeSeries ser)
                             ser.GetAnime().AnimeID);
                         continue;
                     }
+
+                    string confirmationMessage = "";
+                    if (animeListHashtable.ContainsKey(malID))
```

Comment:
I am not sure we can rely on this as MAL list can be cached, there is an open issue on their forums about anime list not always returning latest data.